core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.xul

```diff
@@ -191,7 +191,7 @@
                 </columns>
                 <rows>
                   <row flex="1">
-                    <image id="warning-img-0" src="images/spacer.gif" pen:classname="pentaho-warningbutton"/>
+                    <image id="warning-img-0" src="images/spacer.gif" pen:classname="pentaho-warningbutton" pen:imagealttext="${modeler.warning}"/>
```

Comment:
attribute `pen:imagealttext="${modeler.warning}"`  will be processed by https://github.com/pentaho/pentaho-commons-xul/blob/b72234ced74dc9dc031ea901db8a018030a4009a/gwt/impl/src/main/java/org/pentaho/ui/xul/gwt/tags/GwtImage.java#L111